src/web/wms/src/main/java/org/geoserver/wms/web/WMSAdminPage.html

```diff
@@ -244,6 +244,20 @@
       </ul>
     </fieldset>
 
+    <fieldset>
+      <legend><span><wicket:message key="featuresTransformDisabledTitle"></wicket:message></span></legend>
+      <ul class="choiceList">
+        <li>
+          <input type="checkbox" wicket:id="featuresTransformDisabled"></input>
```

Comment:
"features" is out of context and confusing:
* The flag disables _rendering_ transformations
* It (currently) works only on rasters (but could be implemented for features too, later, although that's going to be harder)

Could it be called "renderingTransformDisabledTitle" instead, and do the same changes in other places that are currently using "featuresTransform"? (e.g., WMSInfo and friends).

Reply:
I changed transformFeatures to transformFeatureInfo.